.github/workflows/build-test.yaml

```
@@ -28,6 +28,16 @@ jobs:
           args: --timeout 10m --exclude SA5011 --verbose --issues-exit-code=0
           only-new-issues: true
 
+  govulncheck:
+    runs-on: ubuntu-latest
+    name: Run govulncheck
+    steps:
```

Is this missing the checkout step?
```suggestion
    steps:
      - name: Checkout code
        uses: actions/checkout@v4
```

---

Just checked the documentation - looks like there's a paral of `repo-checkout` which is true by default! :+1:

---

> govulncheck Github Action accepts several other optional inputs:
> ...
> repo-checkout: checkout the repository, default true
https://github.com/golang/govulncheck-action?tab=readme-ov-file#using-the-govulncheck-github-action

---

https://github.com/jetstack/version-checker/actions/runs/10402581485/job/28807337925?pr=235#step:2:11